README.md

```diff
@@ -214,11 +214,11 @@ By Parsing `mint: false` kepler.gl will keep the component state in the store ev
 
 Read more about [Components][components].
 
-##### `theme` (Object, optional)
+##### `theme` (Object | String, optional)
 
 - default: `null`
 
-Object used to customize Kepler.gl style. Kepler.gl will use the value passed as input to override values from [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
+You can pass theme name or object used to customize Kepler.gl style. Kepler.gl provide an `'light'` theme besides the default 'dark' theme. When pass in a theme object Kepler.gl will use the value passed as input to override values from [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
```

Comment:
do you think we should provide a list of possible values? or a location in the code/docs where they can find values

Reply:
Hmm yea, we should definitely better document this. I am hoping the variable name itself should be self-explanatory.